src/LtiServiceConnector.php

```diff
@@ -67,7 +66,8 @@ public function getAccessToken(array $scopes)
             'form_params' => $authRequest,
         ]);
 
-        $tokenData = json_decode($response->getBody()->__toString(), true);
+        $body = $response->getBody();
```

Comment:
This is the only meaningful change. I haven't been able to test a successful response tho, because I keep getting the "User doesn't belong to this course" error.

Reply:
I can't get past this error either :upside_down_face:

Reply:
```suggestion
        $body = (string) $response->getBody();
```

It seems like from the [Guzzle 7 docs](https://docs.guzzlephp.org/en/stable/psr7.html#body) that the response body needs to be cast to a string first, if we want to pass the body to `json_decode`